docs/pages/concepts/networking/components/network-manager.md

```diff
@@ -1,21 +1,98 @@
+
+But how does the IPAM come into the rescue?
+Liqo enables traffic toward clusters' PodCIDR but does not make any assumption on those address spaces.
+It is legitimate to imagine a situation in which two peered clusters have the same PodCIDR: in this case two Pods could be assigned the same IP addresses, creating a conflict.
+
+{{% notice info %}}
+The PodCIDR is the network used by the CNI for assigning addresses to Pods.
+The ExternalCIDR is the network used by Liqo for assigning addresses to external resources during Reflection(See more [Here](#reflection-of-external-endpoints)).
+{{% /notice %}}
+
+The Liqo solution is to check if networks (PodCIDR and ExternalCIDR) of a foreign cluster are already in use or are reserved in the home cluster (these networks can be specified in the `clusterconfigs.config.liqo.io` resource) and to choose available networks to map them on, if necessary.
+The solution is implemented by the IPAM module, that keeps track of every network used in the local cluster and remaps remote clusters networks that create conflicts (traffic toward these networks will require NAT operation carried out by the [Tunnel Operator](../gateway#tunnel-operator)).
+
+#### IP addresses translation of offloaded Pods
+Liqo enables to offloading of workloads on peered clusters, giving at the same time the administrator the impression Pods are running on its own cluster.
+The [Virtual Kubelet](../../../offloading#virtual-kubelet) is the component in charge of offloading workloads and keep their status always up to date on the original cluster so that the administrator can inspect it.
+However, offloaded Pods are assigned an IP address by the CNI of the foreign cluster, as they actually are run there.
+Even if these addresses are valid on the foreign cluster, they may have no meaning in the original cluster.
+
+Assume that clusters A and B have both PodCIDR equal to 10.0.0.0/24 and that their administrators agree to share resources by establishing a Liqo peering.
+Thus, IPAM module of B decides to map A's PodCIDR on an available network, say 192.168.0.0/24.
+After the peering has been completed, B offloads Pod A1 on cluster A.
+A1 is assigned IP address 10.0.0.34 by the CNI of A, therefore the Virtual Kubelet relative to cluster A running in B detects the IP assignment and would like to update the Shadow Pod Status.
+
+{{% notice info %}}
+The Shadow Pod is a local copy of the Pod resource that Liqo creates and keeps up to date when offloading a Pod.
+{{% /notice %}}
+
+If the Virtual Kubelet updated the Shadow Pod without altering its IP, no response would have received when contacting A1 using the address specified in its Status because that IP address is not valid in cluster B and has to be changed in 192.168.0.34, according to the remapping happened during the peering phase.
```

Comment:
I did not understand this sentence.

Reply:
I meant that without the translation of IP addresses of offloaded pods, entities in the home cluster trying to contact the pod using the address specified in its status would not receive any response.

Reply:
```suggestion
In this particular example, while pod A1 has IP 10.0.0.34, its shadow copy (in cluster B) will have IP 192.168.0.34.
In fact, the former address is valid only in cluster A, while the second is valid only in clyster B; proper NATs, in the data plane, will take care of translating IP src/dest addresses accordingly in order to guarantee direct connectivity from cluster B to its offloaded pod A1.
The translation is driven by VK(A), with the help of IPAM, which determines the proper mapping for involved IP addresses.
```

Reply:
I would move the sentence about the data plane in a Note section, in order to maintain the focus on the translation of addresses in the control plane.
```suggestion
In this particular example, while pod A1 has IP 10.0.0.34, its shadow copy (in cluster B) will have IP 192.168.0.34.
Since it keeps track of how remote networks have been remapped by the home cluster (See #networks-management) , the IPAM module is capable of making the translation of addresses.
For this reason, it provides an API consumed by the Virtual Kubelet when the latter has to update the Shadow Pod Status and therefore its IP address.

{{% notice note %}}
In the data plane the [Tunnel Operator](../gateway#tunnel-operator) will carry out proper NATs operations for translating IP addresses of packets coming from/directed to the offloaded Pod.
{{% /notice %}}
```